ballista/scheduler/src/cluster/storage/sled.rs

```diff
@@ -305,33 +305,33 @@ mod tests {
         Ok(())
     }
 
-    #[tokio::test]
-    async fn multiple_operation() -> Result<(), Box<dyn std::error::Error>> {
-        let client = create_instance()?;
-        let key = "key".to_string();
-        let value = "value".as_bytes().to_vec();
-        let locks = client
-            .acquire_locks(vec![(Keyspace::ActiveJobs, ""), (Keyspace::Slots, "")])
-            .await?;
-
-        let _r: ballista_core::error::Result<()> = with_locks(locks, async {
-            let txn_ops = vec![
-                (Operation::Put(value.clone()), Keyspace::Slots, key.clone()),
-                (
-                    Operation::Put(value.clone()),
-                    Keyspace::ActiveJobs,
-                    key.clone(),
-                ),
-            ];
-            client.apply_txn(txn_ops).await?;
-            Ok(())
-        })
-        .await;
-
-        assert_eq!(client.get(Keyspace::Slots, key.as_str()).await?, value);
-        assert_eq!(client.get(Keyspace::ActiveJobs, key.as_str()).await?, value);
-        Ok(())
-    }
+    // #[tokio::test]
```

Comment:
Is this test intentionally commented out? Should we add a link to an issue explaining why?

Reply:
Sorry, oversight on my part. Commented while trying to get things to compile and forgot to uncomment. Fixed